locust_plugins/listeners.py

```diff
@@ -504,9 +506,9 @@ def user_error(self, user_instance, exception, tb, **kwargs):
             self.function(user_instance, exception, tb, **kwargs)
 
 
-def is_worker():
-    return "--worker" in sys.argv
+def is_worker(env: locust.env.Environment):
```

Comment:
Probably best to just remove the method entirely, now that it is kind of trivial.

Reply:
done